.travis.yml

```diff
@@ -16,14 +16,11 @@ jobs:
       go: "1.13.1"
       install:
         - make goget-tools
-        # binary will be $(go env GOPATH)/bin/golangci-lint
-        - curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.21.0
       script:
         - export PATH="$PATH:$GOPATH/bin"
         - make bin
         - make validate
```

Comment:
Remove ```make validate``` too.

Reply:
I would actually keep it here. It is fairly quick. And there is no harm in running it also here